draft-ietf-webpush-vapid.md

```diff
@@ -366,7 +368,9 @@ need to remember the key that was used when creating a given subscription.
 This authentication scheme is vulnerable to replay attacks if an attacker can
 acquire a valid JWT.  Applying narrow limits to the period over which a
 replayable token can be reused limits the potential value of a stolen token to
-an attacker and can increase the difficulty of stealing a token.
+an attacker and can increase the difficulty of stealing a token.  Placing a
+hard upper limit on the "exp" claim in the JWT allows a push service to reject
+tokens that have excessively long expiration times.
```

Comment:
I'm not entirely convinced about the value of this addition. It's normatively required (the MUST NOT on line 145) and already implied by the sentence directly above ("applying narrow limits"), as it would not make sense if the PS was free to ignore this requirement. It's clear though, so SGTM from that point of view.